src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -352,18 +345,31 @@ public static void init(RegistryEvent.Register<IRecipe> event) {
             ItemStack dustCoal = StackUtils.size(oreDict.get(0), 1);
             RecipeHandler.addCrusherRecipe(new ItemStack(Items.COAL), dustCoal);
         }
+        for (ItemStack dust : oreDict) {
+            RecipeHandler.addEnrichmentChamberRecipe(dust, new ItemStack(Items.COAL));
+            RecipeHandler.addPRCRecipe(dust, new FluidStack(FluidRegistry.WATER, 100), new GasStack(MekanismFluids.Oxygen, 100),
+                    new ItemStack(MekanismItems.OtherDust, 1, 3), new GasStack(MekanismFluids.Hydrogen, 100), 0, 100);
+        }
 
         oreDict = OreDictionary.getOres("dustCharcoal", false);
         if (oreDict.size() > 0) {
             ItemStack dustCharcoal = StackUtils.size(oreDict.get(0), 1);
             RecipeHandler.addCrusherRecipe(new ItemStack(Items.COAL, 1, 1), dustCharcoal);
         }
+        for (ItemStack dust : oreDict) {
```

Comment:
Move inside if statement, see one of my other comments.

Reply:
ok did that.